charts/frontend/templates/network-policy.yaml

```diff
@@ -1,6 +1,6 @@
 # Allow access to drupal or varnish pods from the ingress in the silta-cluster namespace
 {{- $id := "allow-ingress" -}}
-{{- $podSelectors := dict "deployment" "frontend" -}}
+{{- $podSelectors := dict "silta-ingress" "allow" -}}
```

Comment:
Same comment as for the drupal chart, this whole file can now be moved out.

Reply:
fixed now